wire/core/WireInput.php

```diff
@@ -120,6 +120,7 @@ public function __construct() {
 	 *
 	 */
 	public function get($key = '') {
+		if($key === 'requestBody') return $this->requestBody();
```

Comment:
Not sure if this is the best place for this - but didn't feel getVars was correct either.